extracted/plugins/SqueakSSL/src/osx/sqMacSSL.c

```diff
@@ -626,7 +660,7 @@ sqInt sqAcceptSSL(sqInt handle, char* srcBuf, sqInt srcLen, char* dstBuf,
     }
     /* We are connected. Verify the cert. */
     ssl->state = SQSSL_CONNECTED;
-        return SQSSL_OK;
+    return ssl->outLen;

```

Comment:
Naive question: does this not break anything ? ^^

Reply:
Not that I know of.

Doing the following may help clarify this change: using the new version of the SqueakSSL plugin, put `(#result -> result) inspect` before the assignment of `false` to `connecting` in [`#accept` on ZdcSecureSocketStream](https://github.com/svenvc/zodiac/blob/a06165fb935873c6824fa59d24cf3e38bf0c7532/repository/Zodiac-Core.package/ZdcSecureSocketStream.class/instance/accept.st), then try my snippet for setting up a ZnSecureServer.

Assuming it works and you get a ZnResponse with status 200, you should also have gotten a value for `result` greater than 0, so there was a send of `#flushEncryptedBytes:startingAt:count:` in the last iteration of the `#whileFalse:` loop. Without the change, `result` would have been 0, meaning those bytes got lost.